pkg/security/probe/probe_ebpfless.go

```diff
@@ -113,9 +113,11 @@ func copyFileAttributes(src *ebpfless.FileSyscallMsg, dst *model.FileEvent) {
 		dst.PathnameStr = src.Filename
 		dst.BasenameStr = filepath.Base(src.Filename)
 	}
+	dst.IsPathnameStrResolved = true
```

Comment:
I would prefer to use the SetPathnameStr and SetBasenameStr

Reply:
Suggestion applied on last commit